phpBB/phpbb/db/driver/driver.php

```diff
@@ -1037,12 +1037,6 @@ function sql_error($sql = '')
 	function sql_report($mode, $query = '')
 	{
 		global $cache, $starttime, $phpbb_root_path, $phpbb_path_helper;
-		global $request;
-
-		if (is_object($request) && !$request->variable('explain', false))
```

Comment:
Try to replace the old code and change only this line

`if (is_object($request) && !$request->variable('explain', false))`

to 

`if (is_object($request) && !$request->variable('explain', 0))`

Should fix it.

Reply:
This changes are ok IMO, the request is already checked in `phpbb_check_and_display_sql_report()`